vllm/worker/worker.py

```diff
@@ -261,6 +261,11 @@ def execute_model(
             "blocks_to_swap_out": blocks_to_swap_out,
             "blocks_to_copy": blocks_to_copy,
         }
+
+        if num_seq_groups != 0:
+            inputs_to_broadcast, aux = self.model_runner.prepare_inputs_to_broadcast(  # noqa
+                seq_group_metadata_list)
```

Comment:
This will make our formatter work:
```suggestion
            (inputs_to_broadcast, aux) = (self.model_runner.prepare_inputs_to_broadcast(
                seq_group_metadata_list))
```

Reply:
fixed in [ebc7bbf](https://github.com/vllm-project/vllm/pull/5354/commits/ebc7bbf9bf957561864c50fcc0ea39915afc73bf)